lib/ansible/modules/database/postgresql/postgresql_pg_hba.py

```diff
@@ -82,8 +82,10 @@
   order:
     description:
       - The entries will be written out in a specific order.
-      - With this option you can control by which field they are ordered first, second and last.
-      - s=source, d=databases, u=users.
+        With this option you can control by which field they are ordered first, second and last.
+        s=source, d=databases, u=users.
+        This option is deprecated since 2.9 and will be removed in 2.11.
+        Sortorder is now hardcoded to sdu.
```

Comment:
Deprecations should not be backported.  (Although it looks like this is only in the docs, not in code?)

Reply:
Yeah, I understand that. The current implementation basically removes the possibility to control the sort order. Adding it with the changed approach, is not undoable, but does require a big effort.
For what its worth:
1: It is a new module, so nobody has used it up until now, except probably me
2: I'd doubt anyone would ever set sortorder other than the default
3: When I was building the module, I was wondering if anyone would care, but it was easy to add the option. But I'm very sure, it does not make much sense to anyone.

But that does not take away the fact that we would be backporting a depreciation.
So, I see three solutions:
1: Don't backport, leave it broken until 2.9. Not my personal preference. But then again, we have a working module for everybody requiring a workaround.
2: Make an exception.
3: See what we might do to keep the sort order available. I could work on that. But it would push the fix further into the future.
If you want me to work on a working sort order, just let me know, @abadger.
And thanks for your comment.

Reply:
Latest version of this backport works with order option.